bundles/org.openhab.binding.lgwebos/src/main/resources/ESH-INF/i18n/lgwebos.properties

```diff
@@ -38,3 +38,8 @@ actionIncreaseChannelDesc=TV will switch one channel up in the current channel l
 
 actionDecreaseChannelLabel=Channel Down
 actionDecreaseChannelDesc=TV will switch one channel down in the current channel list.
+
+actionShowRCButtonLabel=Remote Control button press
```

Comment:
You have added the labels to support a new action, but I miss changes to class LGWebOSAction as part of this PR

Reply:
Sorry, missed that. I'm on it...

Reply:
OK, please review new commit.

Reply:
looks good to me